packages/api/src/jobs/ai/create_digest.ts

```diff
@@ -832,6 +837,7 @@ export const createDigest = async (jobData: CreateDigestData) => {
         thumbnail: item.libraryItem.thumbnail ?? undefined,
         wordCount: speechFiles[index].wordCount,
         html: summariesInHtml[index],
+        author: item.libraryItem.author ?? '',
```

Comment:
I think its better to leave null/undefined here if its not set. That way i wont do something like `by ${chapter.author}` on an empty item

Reply:
Got it. I have thought double checking with you